src/js/containers/login/LoginContainer.jsx

```diff
@@ -8,11 +8,13 @@ import PropTypes from 'prop-types';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
-import * as LoginHelper from 'helpers/loginHelper';
 import LoginPanel from 'components/login/LoginPanel';
 import LoginMax from 'components/login/LoginMax';
+import Banner from 'components/SharedComponents/Banner';
+import * as LoginHelper from 'helpers/loginHelper';
 import * as sessionActions from 'redux/actions/sessionActions';
 import { kGlobalConstants } from '../../GlobalConstants';
+import * as ReviewHelper from '../../helpers/reviewHelper';
```

Comment:
Did we need to import reviewHelper here? Based on these changes it doesn't seem like we use it.